### PR TITLE
Add pending state in VMVE and VMV

### PR DESCRIPTION
**Pull request by hookak**
In VMVE, all abnormal situations are defined as 'error' state.
However, Cases in which the VMVE could not be restored to the normal state were classified as 'pending' state.

### CODE REVIEW

File: pkg/controller/virtualmachinevolume/virtualmachinevolume_controller_test.go [hunk 1]
```diff
@@ -19,20 +19,20 @@ var _ = Describe("Reconcile", func() {
 		r := createFakeReconcileVmv()
 		_, err := r.Reconcile(reconcile.Request{NamespacedName: testVolumeNamespacedName})
 
-		It("Should return error", func() {
-			Expect(err).ShouldNot(BeNil())
+		It("Should be nil", func() {
```
Comment (hbinkim) [inline pkg/controller/virtualmachinevolume/virtualmachinevolume_controller_test.go:22]: 'Should not return error' sounds better

Comment (hbinkim) [inline pkg/controller/virtualmachinevolume/virtualmachinevolume_controller_test.go:22]: 'Should not return error' sounds better

**Comment by hbinkim** [comment]
LGTM

File: pkg/controller/virtualmachinevolumeexport/virtualmachinevolumeexport_controller.go [hunk 2]
```diff
@@ -95,34 +95,41 @@ func (r *ReconcileVirtualMachineVolumeExport) Reconcile(request reconcile.Reques
...
-		// if destination is local, create local pod and update readytouse to true if it not exist
-		if destination := r.getDestination(); destination == ExporterDestinationLocal {
-			if err := r.syncLocalPod(); err != nil {
-				return err
-			}
+		return reconcile.Result{RequeueAfter: hc.VirtualMachineVolumeExportReconcileAgain}, nil
+	}
+
+	// if there is no pvc, update state to creating and create pvc
+	if err := r.syncExportPvc(); err != nil {
```
Comment (hyoung-90) [inline pkg/controller/virtualmachinevolumeexport/virtualmachinevolumeexport_controller.go:107]: How abount merge following 3 cases?  `syncExportPvc, syncExporterPod, syncLocalPod`

**Review by hbinkim — APPROVED** [review]

File: pkg/apis/hypercloud/v1alpha1/virtualmachinevolumeexport_types.go [hunk 3]
```diff
@@ -40,13 +41,20 @@ const (
 // VirtualMachineVolumeExportState is the current state of the VirtualMachineVolumeExport
 type VirtualMachineVolumeExportState string
 
+const (
+	// VirtualMachineVolumeExportReconcileAgain is time to enter the reconcile loop again
+	VirtualMachineVolumeExportReconcileAgain = 1 * time.Second
```
[the remark this answers is not in the document]
Comment (hbinkim) [reply pkg/apis/hypercloud/v1alpha1/virtualmachinevolumeexport_types.go:46]: How about change constant name to `ReconcileInterval`?

File: pkg/controller/virtualmachinevolume/virtualmachinevolume_controller_test.go (hunk 1 above)
Comment (hookak) [reply pkg/controller/virtualmachinevolume/virtualmachinevolume_controller_test.go:22]: I think 'Should be nil' is more certainty. When the controller enters the 'pending' state, Reconcile 'should return nil'.

File: pkg/apis/hypercloud/v1alpha1/virtualmachinevolume_types.go [hunk 4]
```diff
@@ -24,13 +25,20 @@ type VirtualMachineVolumeSpec struct {
 // VirtualMachineVolumeState is a short string representation of the current VirtualMachineVolume state
 type VirtualMachineVolumeState string
 
+const (
+	// VirtualMachineVolumeReconcileAgain is time to enter the reconcile loop again
+	VirtualMachineVolumeReconcileAgain = 1 * time.Second
```
[the remark this answers is not in the document]
Comment (hookak) [reply pkg/apis/hypercloud/v1alpha1/virtualmachinevolume_types.go]: Thanks, I fixed it!

File: pkg/apis/hypercloud/v1alpha1/virtualmachinevolumeexport_types.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (hookak) [reply pkg/apis/hypercloud/v1alpha1/virtualmachinevolumeexport_types.go]: Thanks for your comments!

File: pkg/controller/virtualmachinevolumeexport/virtualmachinevolumeexport_controller.go (hunk 2 above)
Comment (hookak) [reply pkg/controller/virtualmachinevolumeexport/virtualmachinevolumeexport_controller.go]: Code will be simpler. Thanks!

File: pkg/apis/hypercloud/v1alpha1/virtualmachinevolume_types.go [hunk 5]
```diff
@@ -3,6 +3,7 @@ package v1alpha1
 import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"time"
```
Comment (woohhan) [inline pkg/apis/hypercloud/v1alpha1/virtualmachinevolume_types.go:6]: why added?

**Comment by woohhan** [comment]
LGTM overall but need to pass the test

File: pkg/apis/hypercloud/v1alpha1/virtualmachinevolume_types.go (hunk 5 above)
Comment (hookak) [reply pkg/apis/hypercloud/v1alpha1/virtualmachinevolume_types.go]: Forgot to remove. Thanks!

**Pull request merged**